src/tests/ForecastSummary.test.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { fireEvent, render } from "@testing-library/react";
+import ForecastSummary from "../../components/ForecastSummary";
```

Comment:
The imports in all your tests must have been changed at some point, as all but the app tests dont run. They should all start with '../components/'